Kitodo/src/main/java/de/sub/goobi/forms/LoginForm.java

```diff
@@ -353,20 +350,20 @@ public void setPasswortAendernAlt(String passwortAendernAlt) {
         this.passwortAendernAlt = passwortAendernAlt;
     }
 
-    public String getPasswortAendernNeu1() {
-        return this.passwortAendernNeu1;
+    public String getNewPassword() {
```

Comment:
Your are changing the method then add JavaDoc to this changed method.

Reply:
Do we need javadoc on getters and setters?